src/index.ts

```diff
@@ -103,14 +119,38 @@ export class WebRTCPlayer extends EventEmitter {
     }
   }
 
-  private async onConnectionStats() {
+  private async onConnectionStats() { 
+    
     if (this.peer && this.statsTypeFilter) {
+      let bytesReceivedBlock: number = 0;
       let stats = await this.peer.getStats(null);
+
       stats.forEach((report) => {
         if (report.type.match(this.statsTypeFilter)) {
           this.emit(`stats:${report.type}`, report);
         }
+
+        if (this.detectMediaTimeout == true && report.type.match('inbound-rtp') ) { 
+          this.emit(`stats:${report.type}`, report);
```

Comment:
this event is emitted above so we don't have to emit it twice ( and if not part of the `statsTypeFilter` we shouldn't emit it anyway )